### DISCUSSION TITLE
How can I display 1 item per view and make it center?

### DISCUSSION THREAD

**Discussion by p4rtridge** [asmyshlyaev177/react-horizontal-scrolling-menu#242] [Q&A] [answered]
Great library but I wonder how can I show 1 item show and make it center of menu?

**Comment by asmyshlyaev177** [comment]
Right now i see only one option:
 Set container width and item width manually, also item margin.

Will think more about that.

**Comment by asmyshlyaev177** [comment, marked as answer]
That very hacky solution https://codesandbox.io/s/one-item-j9dzxd?file=/src/index.tsx

**Reply by p4rtridge** [reply]
Thank you. Hope you make this library better
